app/views/RoomActionsView/index.js

```diff
@@ -417,7 +417,7 @@ class RoomActionsView extends React.Component {
 				</Avatar>,
 				<View key='name' style={styles.roomTitleContainer}>
 					{room.t === 'd'
-						? <Text style={[styles.roomTitle, { color: themes[theme].titleText }]} numberOfLines={1}>{room.fname}</Text>
+						? <Text style={[styles.roomTitle, { color: themes[theme].titleText }]} numberOfLines={1}>{room.fname || `${ name }` || 'unnamed'}</Text>
```

Comment:
Add I18n translations to `unnamed`.
Use => `room.fname || name || I18n.t('unnamed')`